draft-ietf-quic-transport.md

```diff
@@ -4196,6 +4416,23 @@ limit mitigates the effect of the stream commitment attack.  However, setting
 the limit too low could affect performance when applications expect to open
 large number of streams.
 
+## Explicit Congestion Notification Attacks
+
+The ECN bits {{!RFC3168}} are an unauthenticated signal from the network. An
+on-path attacker may manipulate the value of the field. Thus, affecting the
```

Comment:
I think this reads more clearly as one sentence.   ie: "An on-path attacker may manipulate the value of the field, affecting the congestion avoidance behavior of the sender."